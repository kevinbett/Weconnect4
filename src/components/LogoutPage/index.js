import React from 'react';
import Navigation from '../Navigation/index';
import LogoutComponent from '../LogoutComponent';

const LogoutPage = () => (

   <div>
       <Navigation />
       <LogoutComponent />
      
   </div>
);

export default LogoutPage;
