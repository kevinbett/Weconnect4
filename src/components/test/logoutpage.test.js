import LogoutPage from '../LandingPage/index'
import { shallow } from "enzyme"

describe('Logout page',() => { 
    it ('Logout page renders', () => { 
        expect(shallow(<LogoutPage/>)).toMatchSnapshot()
    });
} )