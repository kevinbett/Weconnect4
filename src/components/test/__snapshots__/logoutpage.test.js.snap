// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout page Logout page renders 1`] = `
<div>
  <Navigation />
  <Home />
</div>
`;
